patcher.sh

```diff
@@ -340,6 +336,14 @@ main ()
 			HQMUSIC=true
 		fi
 
+		echo "Install AM2R as AppImage?"
```

Comment:
What's the reason for providing this as an option now? I'd personally like for the appimage to remain default for interactive as the "just works" solution.

Reply:
I agree, I initially thought it'd be handy to expose it to end users but it's probably more worthwhile to hide the non-AppImage method as much as possible